### PR TITLE
osx-arm64 manual migration

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin please rerender

**Comment by Tobias-Fischer** [comment]
This is ready for merge @traversaro

**Review by traversaro — APPROVED** [review]

**Comment by traversaro** [comment]
Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ source:
     - unreleased_changes.patch
     - unreleased_changes2.patch
     - fix_python_interface.patch
+    - fix_distutils.patch
 
 build:
   number: 2
```
Comment (traversaro) [inline recipe/meta.yaml]: Perhaps we should bump this? The re-rendering brought a few changes.

**Review by traversaro — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipe/meta.yaml]: Good catch!

**Review by traversaro — APPROVED** [review]

**Review by traversaro — APPROVED** [review]

**Comment by Tobias-Fischer** [comment]
I think you are the only maintainer @traversaro so feel free to merge

**Comment by Tobias-Fischer** [comment]
Fix #3

**Comment by traversaro** [comment]
Sorry, I thought I had added an automerge tag.

**Pull request merged**